nncf/onnx/graph/onnx_graph.py

```diff
@@ -15,10 +15,6 @@
 import onnx
 from onnx import numpy_helper
 
-from nncf.onnx.graph.metatypes.onnx_metatypes import ONNX_OPERATION_METATYPES
-from nncf.onnx.graph.metatypes.onnx_metatypes import WEIGHT_LAYER_METATYPES
-from nncf.onnx.graph.metatypes.onnx_metatypes import OpWeightDef
-
 
 # pylint: disable=too-many-public-methods
 class ONNXGraph:
```

Comment:
My opinion is that ONNXGraph is a redundant abstraction to access ONNX model attributes. I suggest minimizing the use ONNXGraph instead of increasing the use of ONNXGraph to access ONNX model attributes.

Reply:
Yes, good suggestion